transport-native-epoll/src/test/java/io/netty/channel/epoll/EpollSocketTcpMd5Test.java

```diff
@@ -54,7 +54,7 @@ public void setup() {
         server = (EpollServerSocketChannel) bootstrap.group(GROUP)
                 .channel(EpollServerSocketChannel.class)
                 .handler(new ChannelInboundHandlerAdapter())
-                .bind(new InetSocketAddress(0)).syncUninterruptibly().channel();
+                .bind(new InetSocketAddress(NetUtil.LOCALHOST4, 0)).syncUninterruptibly().channel();
```

Comment:
yes.. this is needed as when you use `new InetSocketAddress(0)` it basically creates a wildcard address (0.0.0.0:port). The `EpollSocketTcpMd5Test` expect to have the server bound to ipv4 address.